src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/processedenvelopes/EnvelopeCcdAction.java

```diff
@@ -2,6 +2,7 @@
 
 public enum EnvelopeCcdAction {
     AUTO_ATTACHED_TO_CASE,
+    AUTO_UPDATED_CASE,
     CASE_CREATED,
     EXCEPTION_RECORD;
```

Comment:
Or maybe just `CASE_UPDATED`? I wasn't sure.
There's no AUTO in `CASE_CREATED`...
Let me know what you think.

Reply:
I suggest to rename `CASE_CREATED` to `AUTO_ CASE_CREATED` - later.